### PR TITLE
bindfs: 1.14.8 -> 1.14.9

### PR DESCRIPTION
**Pull request by talyz**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by talyz** [comment]
@GrahamcOfBorg build bindfs

**Comment by lovesegfault** [comment]
Result of `nixpkgs-review pr 110727` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>bindfs</li>
  </ul>
</details>

File: pkgs/tools/filesystems/bindfs/default.nix [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  version = "1.14.9";
   pname = "bindfs";
 
   src = fetchurl {
     url    = "https://bindfs.org/downloads/${pname}-${version}.tar.gz";
-    sha256 = "15y4brlcrqhxl6z73785m0dr1vp2q3wc6xss08x9jjr0apzmmjp5";
+    sha256 = "0fnij365dn4ihkpfc92x63inxxwpminzffyj55krp1w02canpl5n";
   };
 
   dontStrip = true;
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/bindfs/default.nix:12]: Why do we not strip here?

**Comment by SuperSandro2000** [comment]
Changelog

```
2021-01-24  Martin Pärtel <martin dot partel at gmail dot com>

	* Non-zero exit code when unable to chdir into source dir (issue #93).
	* Pass through "-o fsname" to FUSE (issue #94).
	* Released 1.14.9
```

File: pkgs/tools/filesystems/bindfs/default.nix (hunk 1 above)
Comment (talyz) [reply pkgs/tools/filesystems/bindfs/default.nix:12]: Not sure. It's been like that from the start. Seems to work fine without it, though, so I'll remove it.

**Pull request merged**